### ISSUE TITLE
Mass delete not functional in file manager

### ISSUE THREAD

**Issue by parkervcp** [pterodactyl/panel#1614]
**Background (please complete the following information):**
* Panel or Daemon:  Panel
* Version of Panel/Daemon:  v0.7.14
* Server's OS: Debian 9, Docker container runs Alpine 3.9
* Your Computer's OS & Browser:  Fedora 30, Firefox 67

**Describe the bug**
When attempting to mass delete files there is a console error about an uncaught type error.

This is for tracking purposes mainly. Make sure it gets tracked.

```
TypeError: this.sanitizedString is not a functionactions.js:401:51
    deleteSelected actions.js:401
    each jQuery
    deleteSelected actions.js:400
    selectiveDeletion contextmenu.js:74
    jQuery 2
        dispatch
        handle
```

Panel: php 7.3.6

**To Reproduce**
1. Go to the file manager.
2. Select multiple files
3. Click "Mass Delete"

**Expected behavior**
The mass delete popup should show up and let you delete multiple files.

**Comment by Wqrld** [comment]
Can reproduce. i've only pulled the xss and user authentication patches onto 1.7.13 and it broke

**Comment by schrej** [comment]
Could you test the PR by @MatthewSH ?

**Comment by Wqrld** [comment]
That PR worked. Recheck if it still fixes the issue

**Comment by MatthewSH** [comment]
@Wqrld please let me know if the PR exposes any previously reported vulnerabilities.

**Comment by Wqrld** [comment]
Are you sure you want to delete the following files: /<div>cake?

with a filename called <div>cake

Escaping seems to have gone wrong although its not executing anything

**Comment by MatthewSH** [comment]
Okay, I'll be happy to look further into it for the PR. 

So you name a file `<div>cake` correct and it has trouble sanitizing the content?

**Comment by Wqrld** [comment]
Yes @MatthewSH

**Comment by MatthewSH** [comment]
I've attached the following images with two versions of this:
https://i.imgur.com/Cy6rSW8.png
https://i.imgur.com/3a1InIk.png

Are you referring to the div element showing? That's, from what I'm aware of, what it's supposed to do. It's setting the text of an element to the that value, and not rendering it as HTML. I've tried multiple ways to perform some kind of XSS attack via mass delete method and in fact, I can not. If someone who is better can find a way to do it, then be my guest and I can alter PR accordingly.
